pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,42 @@
+url: https://cmap-repos.github.io/cmapplot
+
+home:
+  title: CMAP Themes and Color Palettes
+  description: >
+    Provides themes and color scales for 'ggplot2', based on Chicago
+    Metropolitan Agency for Planning (CMAP) design guidelines.
+
+authors:
+  Chicago Metropolitan Agency for Planning:
+    href: https://www.cmap.illinois.gov
+
+template:
+  params:
+    bootswatch: flatly
+
+reference:
+- title: Theme and Plotting Functions
+- contents:
+  - theme_cmap
+  - finalize_plot
+  - starts_with("geom_")
+  - integer_breaks
+- title: Color Palettes and Gradients
+- contents:
+  - cmap_gradients
+  - cmap_palettes
```

Comment:
`cmap_gradients` and `cmap_palettes` are not functions so much as the lists of colors themselves. Should these, as well as `recessions`, be moved to their own "underlying package data" section? (I am sure there is a better name than that though)

Reply:
> Noel, this looks great! I love the logo--and love that you made it. Also totally content with this theme. I didn't comment too much on the vignettes content, knowing that is to be developed. I presume we do merge this with master, considering you made the PR? I just recall you saying something about the website being implemented from a stand-alone branch...

Yes, let's merge with master. If you look in the gh-pages branch, you'll see the files underpinning the website instead of our package code. The steps to create/update those files are:

1. From the master branch (or, in my case up until now, the pkgdown branch), run `devtools::load_all()` to load the latest package code.
2. Run [`pkgdown::deploy_to_branch()`](https://pkgdown.r-lib.org/reference/deploy_to_branch.html), which modifies the gh-pages branch directly and also pushes it to GitHub, from where the website will automatically reference the latest files.